modules/contracts/apps/calendar/tests/integration.rs

```diff
@@ -83,49 +83,61 @@ fn setup_with_time(
     start_time: Time,
     end_time: Time,
 ) -> anyhow::Result<(
-    Application<Mock, CalendarAppInterface<Mock>>,
-    AbstractClient<Mock>,
+    Application<MockBech32, CalendarAppInterface<MockBech32>>,
+    AbstractClient<MockBech32>,
+    MockBech32,
 )> {
-    let client: AbstractClient<Mock> =
-        AbstractClient::builder(Mock::new(&Addr::unchecked(OWNER)).to_owned())
-            .asset(DENOM, AssetInfoUnchecked::native(DENOM))
-            .build()?;
+    let chain = MockBech32::new("mock");
+    let client: AbstractClient<MockBech32> = AbstractClient::builder(chain.clone())
+        .asset(DENOM, AssetInfoUnchecked::native(DENOM))
+        .build()?;
 
     client.set_balances(vec![
-        ("sender1", coins(INITIAL_BALANCE, DENOM).as_slice()),
-        ("sender2", coins(INITIAL_BALANCE, DENOM).as_slice()),
-        ("sender", coins(INITIAL_BALANCE, DENOM).as_slice()),
+        (
+            chain.create_account("sender1"),
+            coins(INITIAL_BALANCE, DENOM).as_slice(),
```

Comment:
Feels like we want to have `chain.create_account_with_balance()` here?

Reply:
Like so : https://github.com/AbstractSDK/cw-orchestrator/pull/332 ?

Reply:
Yeah, nice!